### PR TITLE
Fade in out animation

### PR DESCRIPTION
**Pull request by elifitch**
Added fade in out animation, suitable for a more subtle pulse effect.  Especially handy for loaders and skeleton screens.

### CODE REVIEW

File: src/animations.css [hunk 1]
```diff
@@ -20,6 +20,7 @@
 .animation-spin { animation: spin 1.5s linear; }
 .animation-fade-in { animation: fadein 1.5s ease-in forwards; }
 .animation-fade-out { animation: fadein 1.5s ease-out reverse forwards; }
+.animation-fade-in-out { animation: fadeinout 1.5s linear forwards; }
```
Comment (samanpwbb) [inline src/animations.css]: Can you add an example to the JSdoc comment so we get something to show up on the docs site?

**Review by samanpwbb — CHANGES_REQUESTED** [review]
Add example for new rule docs then this is goo to go!

File: src/animations.css (hunk 1 above)
Comment (elifitch) [reply src/animations.css]: Jeepers, sure thing

**Comment by elifitch** [comment]
@samanpwbb updated; changed the easing as well, I think this looks a bit less intense, more suitable I think

**Review by samanpwbb — APPROVED** [review]
Looks perfect 👍

**Pull request merged**